data/ai/lua/ai_helper.lua

```diff
@@ -89,11 +89,15 @@ function ai_helper.put_labels(map, cfg)
     --   - keys: (array) if the value to be displayed is a subelement of the LS data,
     --     use these keys to access it. For example, if we want to display data[3]
     --     set keys = { 3 }, if it's data.arg[3], set keys = { 'arg', 3 }
+    --   - no_clear=false: (boolean) if 'true', do not clear existing labels
```

Comment:
I'd rather see this called `clear` rather than dealing in double negatives.

You can still make it default to true by adding a line like `if no_clear == nil then no_clear = true end`.